src/instruction.rs

```diff
@@ -582,6 +661,18 @@ impl Into<String> for ArmInstruction {
             ArmInstruction::Ble { arg1, arg2, target } => {
                 format!("cmp {}, {}\nble {}", arg1, arg2, target)
             }
+            ArmInstruction::Bge { arg1, arg2, target } => {
```

Comment:
In the future, would be better to add a more general way of doing this two instruction hackery for branching. Can you leave a TODO marking this issue?